Abhinav K Gupta/server.js

```diff
@@ -0,0 +1,59 @@
+const express = require('express');
+const ejs = require('ejs');
+const path = require('path');
+const app = express();
+const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
+const session = require('express-session');
+const MongoStore = require('connect-mongo')(session);
+
+const MongoDBURI = process.env.MONGO_URI || 'mongodb://localhost/ManualAuth';

```

Comment:
It's good to see the use of MongoDB here. In the README, mention the technologies that you've used and how you have used them.